dask/dataframe/partitionquantiles.py

```diff
@@ -440,6 +447,25 @@ def dtype_info(df):
     return df.dtype, info
 
 
+def _collapse(partition):
+    return pd.Series(
+        list(partition.itertuples(index=False, name=None)),
+        index=partition.index,
+        name=tuple(partition.columns),
+    )
```

Comment:
Oooo - It may make sense to precede this PR with a simpler PR to support multi-column `sort_columns` using this trick.

@charlesbluca - Note that this approach is not as performant as direct `DataFrame.quantiles`/`DataFrame.searchsorted` support in pandas, but it should "unblock" multi-column sorting :)

Reply:
Yeah this looks nice - thanks for the heads up! I can start up a WIP using this in `sort_values`

Reply:
Cool yeah! I expect this to take a while to work out. There are still some open questions about how things should behave. So  anything that can come out of here and be useful is great!